Source/Web View/WebViewController.swift

```diff
@@ -92,7 +92,7 @@ import THGBridge
  window?.rootViewController = navigationController
  ```
 */
-public class WebViewController: UIViewController {
+public class WebViewController: UIViewController, NSURLSessionDelegate {
```

Comment:
I'd prefer to follow the same pattern of using extensions to conform to delegates much like the extension that implements `UIWebViewDelegate`